sidebar.html

```diff
@@ -0,0 +1,17 @@
+<template id="sidebar-template">
```

Comment:
No need to wrap it in a template, just have the content and we can load it as text